test/network.js

```diff
@@ -292,11 +325,102 @@ describe('Network Test', function() {
     expect(preprocessInfo.getNumberOfChannels()).to.be.a('number');
   });
 
-  it('PreProcessInfo.getNumberOfChannels should throw for invalid argument', () => {
+  it('PreProcessInfo.getNumberOfChannels should throw for invalid argument',
+     () => {
+       const preprocessInfo = network.getInputsInfo()[0].getPreProcess();
+       expect(() => preprocessInfo.getNumberOfChannels(1)).to.throw(TypeError);
+     });
+
+  it('PreProcessInfo.getPreProcessChannel should be a function', () => {
+    const preprocessInfo = network.getInputsInfo()[0].getPreProcess();
+    expect(preprocessInfo.getPreProcessChannel).to.be.a('function');
+  });
+
+  it('PreProcessChannel should throw when the number of channels is 0', () => {
+    const preprocessInfo = network.getInputsInfo()[0].getPreProcess();
+    expect(preprocessInfo.getNumberOfChannels()).to.be.a('number').equal(0);
+    expect(() => preprocessInfo.getPreProcessChannel(0)).to.throw();
+  });
+
+  it('Check the properties of PreProcessChannel', () => {
+    const preprocessInfo = network.getInputsInfo()[0].getPreProcess();
+    preprocessInfo.init(3);
+    const perProcessChannel = preprocessInfo.getPreProcessChannel(0);
+    expect(perProcessChannel).to.be.a('object');
+    expect(perProcessChannel).to.have.property('stdScale');
+    expect(perProcessChannel.stdScale).to.be.a('number');
+    expect(perProcessChannel).to.have.property('meanValue');
+    expect(perProcessChannel.meanValue).to.be.a('number');
+    expect(perProcessChannel).to.have.property('meanData');
+    expect(perProcessChannel.meanData).to.be.a('null');
+  });
+
+  it('PreProcessInfo.getPreProcessChannel should throw for wrong type of argument',
+     () => {
+       const preprocessInfo = network.getInputsInfo()[0].getPreProcess();
+       expect(() => preprocessInfo.getPreProcessChannel('foo'))
+           .to.throw(TypeError);
+     });
+
+  it('PreProcessInfo.getPreProcessChannel should throw for wrong numbers of argument',
+     () => {
+       const preprocessInfo = network.getInputsInfo()[0].getPreProcess();
+       expect(() => preprocessInfo.getPreProcessChannel(1, 2))
+           .to.throw(TypeError);
+     });
+
+  it('PreProcessInfo.setPreProcessChannel should be a function', () => {
     const preprocessInfo = network.getInputsInfo()[0].getPreProcess();
-    expect(() => preprocessInfo.getNumberOfChannels(1)).to.throw(TypeError);
+    expect(preprocessInfo.setPreProcessChannel).to.be.a('function');
   });
 
+  it('PreProcessInfo.setPreProcessChannel should set the stdScale and the meanValue',
+     () => {
+       const preprocessInfo = network.getInputsInfo()[0].getPreProcess();
+       const typedArray1 = new Int8Array(8);
```

Comment:
should be `Float32Array` with size = height * width

Reply:
Done.